client/state/lasagna/socket.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External Dependencies
+ */
+import { Socket } from 'phoenix';
```

Comment:
could we dynamically import this when we actually need it?

Reply:
I ask because this is bloating the main entry point, which nearly everything loads, by about 12KB (compressed). If we can load phoenix on demand, we should be able to get back a chunk of that.

Reply:
Yes! How? 😅 

/me googling and grepping our JS code bases furiously.

Reply:
Try to push the `import { socket } from 'state/lasagna/socket'` import back out of `client/sections-middleware.js`?

Reply:
dynamically import it here instead of statically. remove this line and add 
```
import('phoenix').then( { Socket } => { 
 ...
})
```
down below where you use the Socket

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#Dynamic_Imports

Reply:
Thanks! ICFY still running as I comment but I think we've got that in place successfully now.